src/common/utxobased/engine/makeUtxoEngineState.ts

```diff
@@ -428,58 +428,23 @@ const internalGetFreshAddress = async (
   })
 }
 
-interface ProcessFormatAddressesArgs extends FormatArgs {}
-
-const processFormatAddresses = async (args: ProcessFormatAddressesArgs) => {
```

Comment:
Is that not used?

Reply:
Apparently no longer, removed here: https://github.com/EdgeApp/edge-currency-plugins/commit/0794a357bc0459cb878d53aeabcb801643442689#diff-01dd589479440f34b8666c05923b1c9a88a8446d3a6bdeb058e7795d49eb5fd4L100